<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LicenseDialog</class>
 <widget class="QDialog" name="LicenseDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>License Information</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Geometrize">
      <attribute name="title">
       <string notr="true">Geometrize</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTextBrowser" name="geometrizeLicense">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="GeometrizeLibrary">
      <attribute name="title">
       <string notr="true">Geometrize Library</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QTextBrowser" name="geometrizeLibraryLicense"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Qt">
      <attribute name="title">
       <string notr="true">Qt</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTextBrowser" name="qtLicense"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ChaiScript">
      <attribute name="title">
       <string notr="true">ChaiScript</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextBrowser" name="chaiScriptLicense"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Cereal">
      <attribute name="title">
       <string notr="true">Cereal</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QTextBrowser" name="cerealLicense"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="RapidJSON">
      <attribute name="title">
       <string notr="true">RapidJSON</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QTextBrowser" name="rapidJSONLicense"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="RapidXml">
      <attribute name="title">
       <string notr="true">RapidXml</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QTextBrowser" name="rapidXmlLicense"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="QtGifImage">
      <attribute name="title">
       <string>QtGifImage</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="QTextBrowser" name="qtGifImageLicense"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="GIFLIB">
      <attribute name="title">
       <string>GIFLIB</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_16">
       <item>
        <widget class="QTextBrowser" name="GIFLIBLicense"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="QtGoogleAnalytics">
      <attribute name="title">
       <string>Qt Google Analytics</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QTextBrowser" name="qtGoogleAnalyticsLicense"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="libsimdpp">
      <attribute name="title">
       <string>libsimdpp</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <widget class="QTextBrowser" name="libsimdppLicense"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="FlowLayout">
      <attribute name="title">
       <string notr="true">FlowLayout</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTextBrowser" name="flowLayoutLicense"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ElidedLabel">
      <attribute name="title">
       <string notr="true">ElidedLabel</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QTextBrowser" name="elidedLabelLicense"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="IconAssets">
      <attribute name="title">
       <string>Icons</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QTextBrowser" name="iconAssetsLicense"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TemplateAssets">
      <attribute name="title">
       <string>Templates</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QTextBrowser" name="templateAssetsLicense"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LicenseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LicenseDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
